src/features/mentee-dashboard/services/createApplication.ts

```
@@ -0,0 +1,56 @@
+
+export interface CreateApplicationRequestBody {
+    companyName: string;
+    applyPosition: string;
+    deadline: string;
+    location: string;
+    employmentType: string;
+    careerRequirement: number;
+    url: string;
+}
+
+export interface CreateApplicationResponse {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)